walletlibrary/src/main/java/com/microsoft/walletlibrary/identifier/DidCreator.kt

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved
+
+package com.microsoft.walletlibrary.identifier
+
+import android.util.Base64
+import com.microsoft.walletlibrary.did.sdk.util.Constants
+import com.microsoft.walletlibrary.util.HolderIdentifierCreationException
+import com.microsoft.walletlibrary.util.VerifiedIdExceptions
+import com.nimbusds.jose.jwk.JWK
+import org.erdtman.jcs.JsonCanonicalizer
+
+internal object DidCreator {
+
+    // Creates a DID from Public Key JWK and a DID method. Currently only P-256 keys and did:jwk are supported.
+    fun createDid(jwk: JWK, didMethod: String): String {
+        if (jwk.keyType.value != "EC") {
```

Comment:
Is there no enum that defines this value somewhere? Otherwise, rather than another string literal, can we move this to a const that is reusable?

Reply:
done the same for other string.